governance/docs/what_is_governance.md

```diff
@@ -0,0 +1,61 @@
+
+When documenting your project’s roles, ask the following questions:
+
+* What roles do (or can) project contributors play?
+* What qualifies a person to play a particular role in the project?
+* What duties, privileges, and forms of authority are associated with each role?
+* What project resources are the province or responsibility of people who perform certain roles?
+
+See the [section on Roles]() for more detail and examples of how to develop and document them for your project.
+
+## Policies and Procedures
+
+While role definitions explain how specific contributors participate in the project, there are many activities in a project that involve groups of people, including people external to the project.  Examples of this include things like a release procedure or a contribution process.  These Policies and Procedures (P&P) are what is often thought of as “governance paperwork” for projects.
+
+Some useful questions to ask when developing P&P for your project include:
+
+* How do code and documentation get accepted into the project?
+* How does code get released?
+* What rules govern communication in the project?
+* When do contributors get promoted, and how?
+* How do various decisions, such as technical strategy or project resources, get made?
+* If your project has elections, how are they conducted?
+
+There is a short list of P&P that every CNCF project is expected to have, and a slightly longer list that every project will want to have.  See the [section on Policies & Procedures]() for more detail.
+
+## Accurately Documenting Your Governance
+
+Like technical documentation, governance documentation should explain how things actually work.  If there are aspirational goals, those go in their own section under Roadmap or TODO.  Remember, if you are a Sandbox or Incubating project, you aren’t expected to have all of these things sorted out yet.
+
+It can be tempting to define your project as you would like it to be (or how you would like to present it to the CNCF TOC) rather than how it actually is.  Particularly, project leaders frequently make the mistake of attempting to make the project appear more organized and mature than it actually is, in documentation.  This falls apart when users or contributors expect your project to live up to its governance documentation, and it doesn't.  People who would have been fine with being told a project was single-company at the outset become very upset if they ask for their committer status and are refused later.  For that matter, the CNCF TOC requires projects to be accurate about the things they are still working on, and may penalize projects for not being completely truthful.
+
+*Portions of this text were copied from [The Open Source Way](https://github.com/theopensourceway/guidebook), licensed Creative Commons Share-Alike v4.0*
```

Comment:
Consider adding references to external resources, like the Open Source Way, and https://www.python.org/dev/peps/pep-8002/ (and the governance documents referred from PEP-8002).

Reply:
Hmmm.  We have a separate Resources doc, but point taken to link to the governance section in that doc.

Reply:
Sounds good to me.

Reply:
Added to resources